superset/views/database/api.py

```diff
@@ -262,19 +265,74 @@ def table_metadata(
             500:
               $ref: '#/components/responses/500'
         """
-        table_name_parsed = parse_js_uri_path_item(table_name)
-        schema_parsed = parse_js_uri_path_item(schema_name, eval_undefined=True)
-        # schemas can be None but not tables
-        if not table_name_parsed:
-            return self.response_422(message=_(f"Could not parse table name or schema"))
-        database: Database = self.datamodel.get(pk, self._base_filters)
-        if not database:
-            return self.response_404()
-
+        self.stats_logger.incr(f"init_{self.__class__.__name__}.table")
```

Comment:
Maybe DRY this stats incrementing up a bit? At the moment, there's quite a bit of duplication in terms of reliance on a well-known string format. If maybe just create a func like

```
def inc_stat(self, prefix: str, suffix: str) -> None:
    self.stats_logger.incr(f"{prefix}_{self.__class__.__name__}.{suffix}")
```

Thoughts?

Reply:
Yes, it's a good idea. I'll implement statsd counters on all endpoints on a separate PR